### PR TITLE
feat: added datastream support in shared_vpc_access module

### PR DESCRIPTION
**Pull request by imrannayer**
Added datastream [permission requirements](https://cloud.google.com/datastream/docs/create-a-private-connectivity-configuration) in shared_vpc_access module #787 
Fix `missing profile_id` in test for vpc-sc-project-local test #780

### CODE REVIEW

**Comment by g-awmalik** [comment]
`policy_id` seems to be required variable but is not getting set. Can you take a look at that?

File: test/setup/outputs.tf [hunk 1]
```diff
@@ -58,3 +58,7 @@ output "group_name" {
 output "service_account_email" {
   value = google_service_account.int_test.email
 }
+
+output "policy_id" {
```
Comment (g-awmalik) [inline test/setup/outputs.tf:62]: If we use the gcloud command to evaluate the policy id during the prepare step, we could probably get rid of this.

File: build/int.cloudbuild.yaml [hunk 2]
```diff
@@ -175,3 +175,4 @@ options:
     - 'TF_VAR_gsuite_admin_email=project-factory-test-admin@test.infra.cft.tips'
     - 'TF_VAR_gsuite_domain=test.infra.cft.tips'
     - 'TF_VAR_domain=test.infra.cft.tips.'
+    - 'TF_VAR_policy_id=$_POLICY_ID'
```
[the remark this answers is not in the document]
Comment (imrannayer) [reply build/int.cloudbuild.yaml:178]: We already have following lines in cloudbuild but for some reason it does not work. 
```
- id: converge vpc-sc-project-local
  waitFor:
      - create-all
  name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
  args: ['/bin/bash', '-c', 'source /usr/local/bin/task_helper_functions.sh && export TF_VAR_policy_id=$(gcloud access-context-manager policies list --organization="${TF_VAR_org_id:?}" --format="value(name)") && kitchen_do converge vpc-sc-project-local']
```

[the remark this answers is not in the document]
Comment (g-awmalik) [reply build/int.cloudbuild.yaml:178]: Got it. Ok let's leave the issue open so we can investigate what's happening. I'll merge this for now.

**Review by g-awmalik — APPROVED** [review]

**Pull request merged**